README.md

```diff
@@ -1,24 +1,15 @@
 [![CircleCI](https://circleci.com/gh/LD4P/sinopia_editor.svg?style=svg)](https://circleci.com/gh/LD4P/sinopia_editor)
 [![Coverage Status](https://coveralls.io/repos/github/LD4P/sinopia_editor/badge.svg?branch=master)](https://coveralls.io/github/LD4P/sinopia_editor?branch=master)
 
-# LD4P's BIBFRAME Editor
-
-forked from https://github.com/lcnetdev/bfe
-
-## Overview
-From lcnetdev description: `bfe` is a standalone Editor for the Library of Congress's [Bibliographic Framework
-(BIBFRAME) Initiative][bfi].  It can be used more generically as an editor for RDF data.
-`bfe` uses [BIBFRAME Profiles][profilespec] to render an HTML/UI input form; it is
-capable of integrating 'lookup' services, which query data from external Web APIs;
-and implementers can define the input and extract the output.
-...
-From a design standpoint, the objective with `bfe` was to create the simplest
-'pluggable' form editor one can to maximize experimental implementer's abilities
-to create/edit BIBFRAME data.  The current focus is to transform bfe into a production ready tool.
 
 # Sinopia BIBFRAME Editor
 
-Technical documentation specific to the Sinopia BIBFRAME Editor may also be found in the [wiki](https://github.com/LD4P/sinopia_editor/wiki/Sinopia-Editor).
+Technical documentation specific to the Sinopia BIBFRAME Editor may also be found in the [wiki](https://github.com/LD4P/sinopia_editor/wiki/Sinopia-Editor). The
+[Sinopia Editor][GIT_REPO] is also available in staging at
```

Comment:
looks like you put in some hard returns ... do you want it to be that way, or responsively formated, or?